### ISSUE TITLE
Next ball having a gradient palette as a color crashes the game

### ISSUE THREAD

**Issue by ShamblesSM** [jakubg1/OpenSMCE#76]
To replicate, obtain a Wild Ball or a Fireball (or any sphere with a gradient palette) and swap spheres.

```
src/Shooter.lua:397: attempt to perform arithmetic on field 'colorSpeed' (a nil value)


Traceback:

crash.lua:51: in function 'handler'
src/Shooter.lua:397: in function 'getNextReticalColor'
src/Shooter.lua:285: in function 'draw'
src/Level.lua:769: in function 'draw'
src/Session.lua:93: in function 'draw'
src/Game.lua:189: in function 'draw'
main.lua:104: in function 'draw'
[C]: in function 'xpcall'
```

**Comment by ShamblesSM** [comment]
Created a PR that fixes this issue: #77

**Comment by jakubg1** [comment]
Said PR has been merged, and thus this issue has been resolved.
